docs/docsource/Operation-Architecture/Introduction/supported_clouds.rst

```diff
@@ -15,10 +15,11 @@ the site level to the following supported private and public clouds:
 -  Amazon-S3-based private clouds:
 
    - Scality RING S3 Connector
+   - Red Hat Ceph
```

Comment:
Maybe change this to `Ceph RADOS Gateway`? Don't want user to get the idea that we also support Ceph's block storage

Reply:
Good spot.